experimental/discard-binding.md

```diff
@@ -0,0 +1,15 @@
+# [`void` Discard Bindings for ECMAScript](proposal-discard-binding)
+
+# Patterns
+```js
+interface Void <: Pattern {
```

Comment:
My only concern is that this name could easily be confused as representing any `void` keyword.

What about calling it `VoidPattern` for clarity instead?

Reply:
I am good with that. Previously `Void` was picked because we also have a `Super` node, but since the `super` keyword can only serve for that purpose, `VoidPattern` sounds good to me.